components/accelerator/nvidia/error/xid/component.go

```diff
@@ -5,150 +5,240 @@ package xid
 
 import (
 	"context"
-	"fmt"
+	"database/sql"
+	"sort"
+	"strconv"
+	"sync"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	"github.com/leptonai/gpud/components"
 	nvidia_common "github.com/leptonai/gpud/components/accelerator/nvidia/common"
 	nvidia_component_error_xid_id "github.com/leptonai/gpud/components/accelerator/nvidia/error/xid/id"
-	nvidia_query_nvml "github.com/leptonai/gpud/components/accelerator/nvidia/query/nvml"
 	nvidia_query_xid "github.com/leptonai/gpud/components/accelerator/nvidia/query/xid"
-	"github.com/leptonai/gpud/components/dmesg"
+	"github.com/leptonai/gpud/components/db"
+	os_id "github.com/leptonai/gpud/components/os/id"
 	"github.com/leptonai/gpud/components/query"
 	"github.com/leptonai/gpud/log"
+	pkg_dmesg "github.com/leptonai/gpud/pkg/dmesg"
 )
 
-func New(ctx context.Context, cfg nvidia_common.Config) components.Component {
+const DefaultRetentionPeriod = 3 * 24 * time.Hour
+const DefaultStateUpdatePeriod = 30 * time.Second
```

Comment:
keep same style as others.

```suggestion
const (
       DefaultRetentionPeriod = 3 * 24 * time.Hour
       DefaultStateUpdatePeriod = 30 * time.Second
)
```